op-program/client/mpt/db.go

```diff
@@ -51,10 +51,6 @@ func (p DB) Compact(start []byte, limit []byte) error {
 	return nil // no-op
 }
 
-func (p DB) NewSnapshot() (ethdb.Snapshot, error) {
-	panic("not supported")
-}
-
```

Comment:
Why this change?

Reply:
The `NewSnapshot` method was removed from the DB interface in upstream geth (for each DB type implementation). It's not used in the fault-proof, so we can drop it also.